lib/core/closeDatabaseConnection.js

```diff
@@ -0,0 +1,11 @@
+var debug = require('debug')('keystone:core:closeDatabaseConnection');
+
+module.exports = function closeDatabaseConnection (callback) {
+	this.closeDatabaseConnection = function () {
```

Comment:
You don't need this line at all. `closeDatabaseConnection` is already on the `keystone` prototype, you added it in `index.js`. It's just lines 5:8 that you need in here.